### PR TITLE
[ADD] auth_dynamic_groups

### PR DESCRIPTION
**Pull request by hbrunn**
This is a legacy module I'm busy porting to 8.0

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Same remark than in 7.0.

**Comment by hbrunn** [comment]
thanks, added the snippet from 8.0 (save for the remark about views because this doesn't exist in 6.1)

**Comment by pedrobaeza** [comment]
:+1: 

Travis error is unrelated.

File: auth_dynamic_groups/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+We have a group called `Amsterdam` and want it to contain all users from
+city of Amsterdam. So we use the membership condition
+
+```
+user.partner_id.city == 'Amsterdam'
+```
+
+Now we can be sure every user living in this city is in the right group, and we
+can start assigning local menus to it, adjust permissions, etc.
+    """,
```
Comment (gurneyalex) [inline auth_dynamic_groups/__openerp__.py:54]: @holger can you add Odoo Community Association (OCA) to the authors?

**Comment by gurneyalex** [comment]
Useful module when you need it. :+1:

**Comment by hbrunn** [comment]
> a small unit test would be nice.

will you let this slip if I add a unittest for 8.0? In 6.1, it seems a bit 
pointless to me.

## 

Therp - Maatwerk in open ontwikkeling

Holger Brunn - Ontwerp en implementatie

mail: holger@therp.nl
web: http://therp.nl

**Comment by hbrunn** [comment]
@gurneyalex did the oca part, negotiated about the test part for this branch

**Pull request merged**